packages/charts/src/helpers.js

```diff
@@ -1,9 +1,11 @@
 export function inflateFunctions(config) {
   if (
```

Comment:
The way `inflateFunctions` works with arrays isn't very obvious. Currently, you have to figure out that `targetProperty instanceof Object` will return true also for arrays, and that's how the recursion will work:

```js
if (targetProperty instanceof Object) {
  inflateFunctions(targetProperty);
}
```

I would suggest making the array support more explicit by adding this to the beginning of `inflateFunctions`:

```js
  if (Array.isArray(config)) {
    config.forEach(inflateFunctions);
    return;
  }
```

Reply:
Makes sense. I will update the function.